task1/html/index.css

```diff
@@ -0,0 +1,27 @@
+h1 {
+  text-align: center;
+  color: black;
+  line-height: 100px;
+  margin: 0;
+}
+.container {
+  display: flex;
+  justify-content: center;
+}
+.item {
+  display: flex;
+  justify-content: center;
+  width: 200px;
+  height: 300px;
+  border: 1px solid black;
+  padding: 120px  ;
```

Comment:
let's use padding Only for top+bottom. Left&right will be not needed after centering